### ISSUE TITLE
Is "master" terminology planned to be deprecated? 

### ISSUE THREAD

**Issue by demian711** [opensearch-project/helm-charts#276]
Sorry if this is not the place.
Reading Opensearch 2.0 documentation, says that inclusive language is being implemented. However for helm charts "old" terminology is still being used.

Do you plan to deprecate this in a planned version?

thanks

**Comment by TheAlgo** [comment]
Good point, we can plan this out. This is a good change to have. What do you say @prudhvigodithi @peterzhuamazon @DandyDeveloper ?

**Comment by peterzhuamazon** [comment]
@demian711 @TheAlgo please follow this meta issue on the status of inclusive language changes:
https://github.com/opensearch-project/OpenSearch/issues/2589#issuecomment-1101707941

Thanks.

**Comment by TheAlgo** [comment]
@peterzhuamazon So I assume we can deprecate the master terminology for helm charts in for charts which are using 2.x version as default version?

**Comment by peterzhuamazon** [comment]
> @peterzhuamazon So I assume we can deprecate the master terminology for helm charts in for charts which are using 2.x version as default version?

The problem being not everyone has deprecated the terminology.
We can try from our part but IDK if that would cause any issues if any plugin still uses master node terminology.

Thanks.
